Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocList.css

```diff
@@ -373,15 +373,26 @@ div.bocList div.bocListTableBlock div.bocListNavigator a:not([href]) img
 /*endregion*/
 
 /*region validation marker*/
+div.bocList .bocListCellStructureElement > .validationErrorMarker,
+div.bocList .bocListContent > .validationErrorMarker
+{
+  border: var(--remotion-themed-border-placeholder);
+  border-radius: var(--remotion-themed-border-radius);
+}
+
 div.bocList .bocListCellStructureElement > .validationErrorMarker
 {
-  margin-top: calc(var(--remotion-themed-spacing) / 2);
+  /* .bocListContent already comes with padding so we only need this for .bocListCellStructureElement */
+  padding: calc(var(--remotion-themed-spacing) / 2);
```

Comment:
`padding: calc(var(--remotion-themed-spacing-small) - var(--remotion-themed-border-width));`

I believe this version creates a more precise alignment. Only, it doesn't applies to the validation column icon itself. Could you please check if/wat options we have?

(Without this, the validation icosn are 1px below the selection radio button and the top border of the selection checkbox.

Reply:
Even with your suggestion the initial problem still exists. The problem is unrelated to this issue but rather a rounding problem that causes the checkbox to move slightly depending on where it is positioned on the screen. If you remove the error messages from the BocList above (see screenshot below) then the positioning is correct again.

![image](https://github.com/re-motion/Framework/assets/5082673/b3743484-593a-4daf-b96e-48e86771926f)

Reply:
I'm not convinced. we're adding a 1px border and a padding. in other places I believe add a padding minus the border plus a separate border space. This results in a 1px offset.

Reply:
The diff makes it seem like the border is new but it isn't. If you look at the bottom left of the diff you can see that the "placeholder" border always existed. The border isn't new and the added padding replaces the `margin-top` so there should not be a change in the positioning of the markers. Even on develop the checkboxes and markers were misaligned by one pixel in the body.

I think that we should look at this problem in a separate issue as there is a new interaction that makes it clearer that this is a rounding problem that causes the slight misalignment. Without validation markers we get the following alignment:

![image](https://github.com/re-motion/Framework/assets/5082673/fe3701ba-824e-4570-b65e-6b8974cc8a59)

The top of the checkbox and marker align perfectly. Now here is the screenshot with validation errors enabled:

![image](https://github.com/re-motion/Framework/assets/5082673/33cd421e-6ee5-43af-aad9-c7e7be32bcf5)

We can see that the all the markers aligned correctly to each other and to the text baseline. But the checkbox is now a pixel above the marker. In fact, the checkbox did not move but rather the text and marker moved down a pixel. To show that this is a rounding error here is a screenshot with the form grid validation messages from the top BocList removed (see my other comment above):

![image](https://github.com/re-motion/Framework/assets/5082673/e8d4fd3e-f42e-4adc-a357-5e086fc8bded)

Without any CSS changes the alignment is fixed again, just because the location of the bottom BocList has changed. I think that the changes here are good as proposed but that we have to look into the positioning in general if we want to fix the 1px offset in the checkbox. But I would do that as part of a separate issue.